src/overlays/actors/ovl_En_Ex_Item/z_en_ex_item.h

```diff
@@ -15,7 +15,7 @@ typedef struct EnExItem {
     /* 0x0150 */ s16 getItemObjId;
     /* 0x0152 */ s16 type;
     /* 0x0152 */ s16 unusedParam;
-    /* 0x0156 */ s16 drawItemId;
+    /* 0x0156 */ s16 drawId;
```

Comment:
Imo, `drawId` is too general outside of the context of get items, so stuff like actor struct fields should probably include `getItem` (or `gi` if we want to keep it short)